accessibility_api/accessibility_lib/scripts/commands.py

```diff
@@ -0,0 +1,65 @@
+'''
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this file,
+You can obtain one at http://mozilla.org/MPL/2.0/.
+'''
+
+from accessibility_api.accessibility_lib.scripts.accessible import (
+    accessible, interface_ptr_types
+)
+from accessibility_api.accessibility_lib.scripts.constants import (
+    SUCCESSFUL_RESPONSE, ERROR_RESPONSE
+)
+
+
+def execute_command(params):
+    """Execute command on accessible object and returns value"""
+
+    cmd = params.get('function')
+    function_params = params.get('param')
+    value = None
+
+    acc_obj = accessible(params).serialize_result(0)
+    if acc_obj.get('status') == ERROR_RESPONSE:
+        return {
+            'status': ERROR_RESPONSE,
+            'json': None
+        }
+    _json = acc_obj.get('json')
+
+    # Get accessible field from JSON
+    if cmd in _json:
+        value = _json[cmd]
+    # Call accessible method
+    else:
+        # Localize paramaters
+        localized_params = []
```

Comment:
I think we should avoid reassigning. If you move function_params into else, see if you can just populate in the for loop and just iterate straight on params.get('param')